modules/ppcp-wc-gateway/src/Helper/CheckoutHelper.php

```diff
@@ -35,14 +35,6 @@ public function is_checkout_amount_allowed( float $minimum, float $maximum ): bo
 			if ( $cart_total < $minimum || $cart_total > $maximum ) {
 				return false;
 			}
-
-			$items = $cart->get_cart_contents();
-			foreach ( $items as $item ) {
-				$product = wc_get_product( $item['product_id'] );
-				if ( is_a( $product, WC_Product::class ) && ! $this->is_physical_product( $product ) ) {
-					return false;
```

Comment:
If it was needed for another gateway, is it handled somewhere else now?

Reply:
@AlexP11223 good point, fixed!